packages/tasks/src/hardware.ts

```diff
@@ -0,0 +1,305 @@
+/**
+ * Biden AI Executive Order
+ * https://www.whitehouse.gov/briefing-room/presidential-actions/2023/10/30/executive-order-on-the-safe-secure-and-trustworthy-development-and-use-of-artificial-intelligence/
+ */
+export const TFLOPS_THRESHOLD_WHITE_HOUSE_MODEL_TRAINING_TOTAL = 10 ** 14;
+export const TFLOPS_THRESHOLD_WHITE_HOUSE_MODEL_TRAINING_TOTAL_BIOLOGY = 10 ** 11;
+export const TFLOPS_THRESHOLD_WHITE_HOUSE_CLUSTER = 10 ** 8;
+
+/**
+ * EU AI Act
+ * https://ec.europa.eu/commission/presscorner/detail/en/qanda_21_1683
+ */
+export const TFLOPS_THRESHOLD_EU_AI_ACT_MODEL_TRAINING_TOTAL = 10 ** 13;
```

Comment:
This part feels a bit weird here, at least without context of how it's used

Reply:
haha yes it's mostly for fun, if we want to add a `x-risk` label on GPU rich users' profiles at some point =)